src/disaster_vehicle.cpp

```diff
@@ -300,7 +301,9 @@ static bool DisasterTick_Zeppeliner(DisasterVehicle *v)
 	}
 
 	if (IsValidTile(v->tile) && IsAirportTile(v->tile)) {
-		Station::GetByTile(v->tile)->airport.blocks.Set(AirportBlock::RunwayIn);
+		Station *st = Station::GetByTile(v->tile);
+		st->airport.blocks.Set(AirportBlock::Zeppeliner);
+		st->airport.blocks.Set(AirportBlock::RunwayIn);
```

Comment:
```suggestion
		st->airport.blocks.Set({AirportBlock::Zeppeliner, AirportBlock::RunwayIn});
```
The intermediate variable `st` could be also removed.